### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by bovirus**
Update Italian translation

### CODE REVIEW

File: translations/copyq_it.ts [hunk 1]
```diff
@@ -271,176 +271,176 @@ Potete impostare il comando nelle preferenze.</translation>
...
         <source>CopyQ Add Commands</source>
-        <translation type="unfinished"></translation>
+        <translation>Comandi Aggiungi CopyQ</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="81"/>
         <source>Ctrl+Shift+1</source>
         <comment>Global shortcut for some predefined commands</comment>
-        <translation type="unfinished"></translation>
+        <translation>CTRL+Miausc+1</translation>
```
Comment (hluk) [inline translations/copyq_it.ts:280]: Seems that this shortcut is not recognized by the Qt GUI framework. When I try to map the shortcut in the app it still shows as "Ctrl+Shift+1" so it should probably be the same in the translation.

The other translated shortcut ("CTRL+SPAZIO") works fine.

BTW, you can test the translation on Windows by clicking on details link next to "continuous-integration/appveyor/pr — AppVeyor build succeeded" (after the app is successfully built)

> image

clicking on the first job name

> image

selecting ARTIFACTS and downloading portable app in zip file.

> image

File: translations/copyq_it.ts [hunk 2]
```diff
@@ -271,176 +271,176 @@ Potete impostare il comando nelle preferenze.</translation>
...
         <location filename="../src/gui/addcommanddialog.cpp" line="177"/>
         <source>Paste as Plain Text</source>
-        <translation type="unfinished"></translation>
+        <translation>Incolla come testo semplice</translation>
     </message>
     <message>
         <location filename="../src/gui/addcommanddialog.cpp" line="183"/>
         <source>Shift+Return</source>
-        <translation type="unfinished"></translation>
+        <translation>Maiusc+INVIO</translation>
```
Comment (hluk) [inline translations/copyq_it.ts:380]: Same as above, this is recognized only as `Shift+Invio` (though letter-case doesn't matter).

**Comment by hluk** [comment]
I can actually merge this right now, if you can fix the shortcuts later in different commit, there are possibly broken shortcuts in other translations too (though this is the way to translate shortcuts described in Qt documentation/examples).

**Comment by bovirus** [comment]
Could be possible because should be "CTRL+Maiusc+1"

I found that it's not finalized translation and shouldn't be important teh content of translation.

I pushed also the update for .iss installer script.

Please rebuild the installer to check the Italian translation in place,.

**Comment by hluk** [comment]
> Could be possible because should be "CTRL+Maiusc+1"

It doesn't work either. Qt translates Shift as Shift -- see [qttranslations/translations/qtbase_it.ts](https://github.com/qt/qttranslations/blob/e51b921f6fa0fb64a8a7f4159024d1a2f1872772/translations/qtbase_it.ts#L5722).

> Please rebuild the installer to check the Italian translation in place,.

Translation in installer looks OK.

**Comment by hluk** [comment]
@bovirus Any news on this PR? I can merge it and fix the shortcuts later.

Note that it can be difficult to merge after someone else updates the translation.

**Comment by bovirus** [comment]
I can fix it after 29.08.

**Comment by hluk** [comment]
@bovirus OK, thanks a lot.

**Comment by hluk** [comment]
@bovirus Any update? Should merge and fix the shortcut?

**Comment by bovirus** [comment]
I will upload the changes this evening.

**Comment by bovirus** [comment]
Hi.
I made a new pull request for Italian translation update.,
Thanks.

**Comment by hluk** [comment]
> I made a new pull request for Italian translation update.

Thanks. Should I close this pull request? I don't see changes for "shared/copyq.iss" in the new PR.

**Comment by bovirus** [comment]
I just posted a new file.

Please check.

**Comment by hluk** [comment]
I still see [changes only in one file](https://github.com/hluk/CopyQ/pull/794/files) in the new PR.

**Comment by bovirus** [comment]
As you can see a fixed the "Shift" issues and add new translations for many untranslated strings.
I believe that the file is OK.

**Comment by hluk** [comment]
OK, so should I close this PR and leave "shared/copyq.iss" without changes? (Because the other PR contains modifications only for "translations/copyq_it.ts" file.)

**Comment by bovirus** [comment]
I believ that I can have some problems with github commands.
You can close the PR:
If you are agree in the future I will contact in advance and send the new file to you to load in the repository. Thanks.

**Comment by hluk** [comment]
@bovirus No problem. I think the best option for translating is to use [Weblate](https://hosted.weblate.org/projects/copyq/). Though not everything can be translated through it (setup strings) and some people prefer other apps.

**Comment by bovirus** [comment]
Hi.

I'm using the web interface as you suggested (I ebelieve that it's easier).

What's the relation between translation status in GitHub and on the website?
Are always synchronized?

What about when I cannot use the same accelerator (string with &) as in the source text (ex. because the same letter is not used in translation)?

Can I use another accelerator or I have to leave the translated text without accelerator?

Please take care that at the end of translation could be necessary a proof check because in Italian there are a lot of male/female strings that I cannot check before to run the program with translation in place.

Thanks.

**Comment by hluk** [comment]
> What's the relation between translation status in GitHub and on the website?
Are always synchronized?

The status image is taken from Weblate so it should be more or less up-to-date. But sometime caching on web browser or on a server gives bit older version of the status.

> Can I use another accelerator or I have to leave the translated text without accelerator?

You should use unique accelerators (can differ from the original text). This is really difficult because you don't see the whole context with the text.

I've noticed Chinese translation having text and original accelerator in parentheses, e.g. `格式(&M):` for `For&mat:`.

You can leave it empty if it's not easy to figure out the accelerator. I think it would be better it this is handled in code (some Qt themes actually do this automatically).

> Please take care that at the end of translation could be necessary a proof check because in Italian there are a lot of male/female strings that I cannot check before to run the program with translation in place.

OK, same with Czech. :)

BTW, I'll try to update [documentation for translating](http://copyq.readthedocs.io/en/latest/translations.html) so let me know if you have any ideas.

**Comment by bovirus** [comment]
I checked that for website translation you are using is Weblate.
I checked on github Weblate repository that Italian translation (for interface) contain several errors.
i can see some of theese errors in your website translation interface.
Can i send you the .po files (djuango.po/djangos.po) to update your translationw ebsite?
Thanks.

**Comment by hluk** [comment]
@bovirus Not sure what you mean. Which errors are you referring to? Weblate is separate project (not owned by me), CopyQ is only using it as translation service.

**Comment by hluk** [comment]
@bovirus Translations on Weblate are not reflected immediately in this repository. Weblate pushes the changes only after some time so as not to create too many commits.

**Comment by hluk** [comment]
@bovirus I just merged changes from Weblate manually.

**Comment by bovirus** [comment]
I just mentioned that you are using Weblate for your translation project.

The interface of Weblate (on Copyq translation project) contains some errors about Italian translation (about Weblate interface- not CopyQ) becuase the django/djangops po file (got froM Weblate githubr epository) are not right.
.
I update thE strings into Weblate github repository and I sent to your email the .PO file t update them manually in your Weblate site that handle your CopyQ translation project.

 I keep informed you about Italian translation status for Weblate project.
Thanks :-)

**Comment by hluk** [comment]
@bovirus OK, thanks! (As I mentioned, there is no need to send any translation files from Weblate next time, it will push all the changes eventually.)

**Pull request merged**